unlock-wordpress-plugin/inc/classes/fullpostpage/index.php

```diff
@@ -0,0 +1,2 @@
+<?php
+// Silence is golden.
```

Comment:
:D

Reply:
Lol... its a security measure to ensure the folder I added is not accessible directly from the browser. But seems you don't like it there. Have removed it completely considering is not part of the features been implemented in the issue..